src/lib/components/submit/SubmitSteps.svelte

```diff
@@ -78,28 +84,50 @@
 		Sign-in
 	</Step>
 
-	<Step status={selectStatus}>
-		<svelte:fragment slot="step">2</svelte:fragment>
-		Select
-	</Step>
+	{#if $showSelect}
```

Comment:
Not sure why you are duplicating code here? Isn't just one step that should be added or not? This seems difficult to maintain